### PR TITLE
Feat: added contributing guidelines and fixed linting commands

### PR DESCRIPTION
**Pull request by xelacast**
Updates issue #89 

[x] The PR title follows the conventions in [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
[x] Ran a functional test before committing

I added the CONTRIBUTING.md file to the root dir to walk through the contributions.
Fixed lining command issues in package.json ```pnpm``` lint and ```pnpm lint:fix``` and found unbreaking errors in many files.

This is still a work of progress and I don't know if we should use the changeset yet for versioning. An issue with the changeset is committing the summaries to the repo. We can offload these somewhere else so it does not bloat the repo. A work in progress.

TODO:
[ ] - ~~Offload changeset if applicable in the current timeframe~~ (Implement when we need it)
[ ] - ~~Create GitHub actions for linting and formatting checks for PRs~~ For the next contributing PR
[ ] - ~~Create GitHub actions for build time errors~~ For the next contributing PR

Feedback would be appreciated 👍

### CODE REVIEW

File: .changeset/config.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+  "changelog": ["@changesets/cli/changelog", { "repo": "FlowTestAi/FlowTest" }],
```
Comment (jsajal) [inline .changeset/config.json:3]: nit: `FlowTestAi/FlowTest` -> `FlowTestAI/FlowTest`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+```bash
+git add <file> && git commit -m "feat/fix/chore/docs: commit message"
+```
+
+### When you're done
+
+Please make a manual, functional test of your changes.
+
+If your change should appear in the changelog, i.e. it changes some behavior of either the CLI or the outputted application, it must be captured by `changeset` which is done by running
```
Comment (jsajal) [inline CONTRIBUTING.md:68]: `outputted` -> `electron`

also can add a line for case when contributor does not want the hassle of changelog
`....If not then skip to the pr creation step`

File: package.json [hunk 3]
```diff
@@ -12,11 +12,10 @@
...
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "postinstall": "cd packages/flowtest-electron && pnpm install",
-    "lint": "eslint src/**/*.{js,jsx,ts,tsx,json}",
-    "lint:fix": "eslint --fix 'src/**/*.{js,jsx,ts,tsx,json}'",
+    "lint": "eslint . --ext .js,.json,.cjs",
+    "lint:fix": "eslint --fix . --ext .js,.json,.cjs",
     "format": "prettier --write '**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc",
-    "clean": "rm -rf node_modules/ && rm -rf packages/flowtest-electron/node_modules/"
+    "clean": "rm -rf node_modules/ && rm -rf packages/flowtest-electron/node_modules/ && rm -rf packages/flowtest-cli"
```
Comment (jsajal) [inline package.json:18]: I think you mean `rm -rf packages/flowtest-cli/node_modules/`

we don't want to clean the cli itself 😄

**Review by jsajal — CHANGES_REQUESTED** [review]
Left some comments but overall this is fantastic. 

Let me address some of the points that you raised

`linting errors` : yeah we are aware about them, I will find some time to fix those. Can you actually file an issue? that way it will be logged and it can be worked upon much faster.

> Offload changeset if applicable in the current timeframe

I think it's fine for now, if we see changelogs file increasing a lot we can always make more frequent minor releases (that deletes all changelog files till that point)

> [ ] - Create GitHub actions for linting and formatting checks for PRs
> [ ] - Create GitHub actions for build time errors

This is super helpful especially the second action about build and build time. I would prioritize that first.

File: .changeset/config.json (hunk 1 above)
Comment (xelacast) [reply .changeset/config.json:3]: Good catch

File: CONTRIBUTING.md
Comment (xelacast) [inline CONTRIBUTING.md:1]: Make sure to add a brief description about the eco system

Comment (jsajal) [reply CONTRIBUTING.md:1]: oh yeah, i forgot to mention that in the comments, yes i will add and merge the PR later this weekend

Comment (xelacast) [reply CONTRIBUTING.md:1]: Sick! That comment was meant for me lol. I was going to add a brief description of what the folders are doing/holding but you would be more equipped for that task.

**Review by jsajal — APPROVED** [review]

**Pull request merged**